web-sdk/v3.md

```diff
@@ -628,3 +628,156 @@ Here are some examples of what is achievable using the customization:
+  </tr>
+  <tr>
+    <td><code>accountId</code></td>
+    <td>string</td>
+    <td>Account ID of the program where the card was enrolled</td>
+  </tr>
+  <tr>
+    <td><code>programId</code></td>
+    <td>string</td>
+    <td>ID of the program where the card was enrolled</td>
+  </tr>
+  <tr>
+    <td><code>scheme</code></td>
+    <td>string<br>Possible values: 'mastercard', 'visa', 'amex'</td>
+    <td>Card scheme</td>
+  </tr>
+  <tr>
+    <td><code>lastNumbers</code></td>
+    <td>number</td>
+    <td>Last 4 numbers of the card</td>
+  </tr>
+  <tr>
+    <td><code>firstNumbers</code></td>
+    <td>number</td>
+    <td>First 6 numbers of the card</td>
+  </tr>
+  <tr>
+    <td><code>expYear</code></td>
+    <td>number</td>
+    <td>First 6 numbers of the card</td>
+  </tr>
+  <tr>
+    <td><code>expYear</code></td>
+    <td>Number</td>
+    <td>Expiration year of the card</td>
+  </tr>
+  <tr>
+    <td><code>expMonth</code></td>
+    <td>Number</td>
+    <td>Expiration month of the card</td>
+  </tr>
+  <tr>
+    <td><code>expDate</code></td>
+    <td>String</td>
+    <td>Expiration date (MM/YY format)</td>
+  </tr>
+  <tr>
+    <td><code>created</code></td>
+    <td>string</td>
+    <td>Card enrollment date</td>
+  </tr>
+  <tr>
+    <td><code>countryCode</code></td>
+    <td>String</td>
+    <td>Card country code</td>
+  </tr>
+  <tr>
+    <td><code>termsOfUse</code></td>
+    <td>Boolean</td>
+    <td>Accepted or declined terms of use</td>
+  </tr>
+  <tr>
+    <td><code>live</code></td>
+    <td>Boolean</td>
+    <td>Flag that checks if the card is from a live or a test environment</td>
+  </tr>
+  <tr>
+    <td><code>metadata</code></td>
+    <td>Object</td>
+    <td>Returns the metadata set by the enrollment API call</td>
+  </tr>
+</tbody>
+</table>
+
+## Error object
+
+This object is returned whenever you receive an error callback.
+
+<table>
+<thead>
+  <tr>
+    <th>Parameter</th>
+    <th>Value</th>
+    <th>Description</th>
+</thead>
+<tbody>
+  <tr>
+    <td><code>cardId</code></td>
+    <td>string (optional)</td>
+    <td>ID of the enrolled card. Might not be present if card is not enrolled at the error stage</td>
+  </tr>
+  <tr>
+    <td><code>date</code></td>
+    <td>string</td>
+    <td>Date when the error happened</td>
+  </tr>
+  <tr>
+    <td><code>type</code></td>
+    <td>string</td>
+    <td>Returns the error <a href="#error-codes">code</a></td>
+  </tr>
+  <tr>
+    <td><code>message</code></td>
+    <td>string</td>
+    <td>Returns the error message</td>
+  </tr>
+</tbody>
+</table>
+
+### Error codes
+
+<table>
+<thead>
+  <tr>
+    <th>Value</th>
+    <th>Description</th>
+</thead>
+<tbody>
+  <tr>
+    <td>service-unavailable</td>
+    <td>Service unavailable at the moment</td>
+  </tr>
+  <tr>
+    <td>item-exists</td>
+    <td>Card already exists</td>
+  </tr>
+  <tr>
+    <td>invalidProgramId</td>
+    <td>Invalid program ID</td>
+  </tr>
+  <tr>
+    <td>credential</td>
+    <td>Invalid SDK key</td>
```

Comment:
We have some other error ids. 
Complete list:
| Already in Doc | Enum                     | Value (just for reference not part of doc) |
|----------------|-------------------|-------|
| ✓                       | serviceUnavailable | service-unavailable | 
|  ✓                          | cardAlreadyExists  | map.already.exists'| 
|                           | invalidCardId         | invalidProgramID | 
|  ✓                       | invalidSDKKey        | credential | 
|  ✓                        | inexistentProgram | item-exists | 
|                           | invalidProgramID | uuid | 
| | unauthorized | Unauthorized | 
| | incorrectAmount | verification-incorrect-amount | 
| | maximumAttemptsReached | verification-max-attempts-reached | 
| | verificationNotFound | verification-not-found | 
| | cardAlreadyVerified | card-already-verified | 
| | cardNotFound | card-not-found | 

| | unableToVerify | verification-error-generic | 


For Reference: 
1. [Card Enroll Errors](https://github.com/FidelLimited/web-sdk/blob/develop/src/types/card-enroll-error.ts)
2. [Card Verify Errors](https://github.com/FidelLimited/web-sdk/blob/develop/src/types/card-verify-error.ts
)

Reply:
I'm not sure if it's worth splitting by flow (verification or enrollment) at this stage. I tend to prefer a single table.

Reply:
@murtas thanks for remind me of this. I was too focused on the card enrollment that ended up forgetting about other stuff.
Just a question, which one should we consider as invalidProgramId? We have `uuid` and `invalidProgramId`.

Another observation, the value column is the one that actually goes to the callback function, and the enum is the one that we use in our code (or I might be missing something).

About the split, I would keep it as the same table for now, the errors are very intuitive IMO

Reply:
> Another observation, the value column is the one that actually goes to the callback function, and the enum is the one that we use in our code (or I might be missing something).


Correct.

> Just a question, which one should we consider as invalidProgramId? We have uuid and invalidProgramId.

uuid, updated the table accordingly.